src/goto-analyzer/goto_analyzer_parse_options.cpp

```diff
@@ -810,8 +825,9 @@ void goto_analyzer_parse_optionst::help()
     "\n"
     "Abstract interpreter options:\n"
     // NOLINTNEXTLINE(whitespace/line_length)
-    " --location-sensitive         use location-sensitive abstract interpreter\n"
-    " --concurrent                 use concurrency-aware abstract interpreter\n"
+    " --location-sensitive         one domain per location\n"
```

Comment:
What happens if neither location-sen nor location-insen is supplied?

Reply:
It defaults to location-sensitive ( https://github.com/diffblue/cbmc/blob/develop/src/goto-analyzer/goto_analyzer_parse_options.cpp#L246 ) as the "expected" style of abstract interpretation.

Reply:
Mark the default in the help text in that case

Reply:
domain -> abstract state,  please

Reply:
Maybe put options and help text into defines so that they can be reused in janalyzer